Packet++/header/WireGuardLayer.h

```diff
@@ -0,0 +1,252 @@
+#pragma once
+
+#include "Layer.h"
+#include "IpAddress.h"
+#include "MacAddress.h"
+
+/// @file
+
+/**
+ * \namespace pcpp
+ * \brief The main namespace for the PcapPlusPlus lib
+ */
+namespace pcpp
+{
+
+	/**
+	 * WireGuard message types
+	 */
+	enum WireGuardMessageType
```

Comment:
Minor thing, could you change that to `enum class`. We have been trying to use `enum classes` going forward, and porting an existing `enum` to `enum class` is a pain after it is already in use while keeping backwards compatibility.

Reply:
Thanks for your feedback.